libmscore/score.cpp

```diff
@@ -71,6 +71,8 @@
 #include "breath.h"
 #include "instrchange.h"
 
+#include <limits>
```

Comment:
Isn't this more a candidate for all.h?  `#include <limits.h>` is in there already